lms/static/sass/course/courseware/_courseware_search.scss

```diff
@@ -85,6 +85,13 @@
       bottom: lh(.75);
       text-transform: uppercase;
     }
+    .search-results-ellipsis {
+      &:after {
+        content: '\2026';
+      }
+      font-size: smaller;
```

Comment:
NIT: extend something from the LMS _typography.scss file instead of using font-sizes directly in the styling. %t-copy-sub2 might be the size you're looking for, but you can try out others to see what is closer.